<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQRWidget</class>
 <widget class="QWidget" name="CQRWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CQRWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget_main
{
border-image: url(:/Image/Resources/Image/img_bg.png);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_main" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_title" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>95</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>95</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>25</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>25</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_back">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">#btn_back
{
min-width:55px;
max-width:55px;
min-height:50px;
max-height:50px;
border-radius: 10px;
}
#btn_back{border-image: url(:/Image/Resources/Image/btn_Return_nor.png);}
#btn_back:hover{border-image:url(:/Image/Resources/Image/btn_Return_sele.png);}
#btn_back:pressed{border-image:url(:/Image/Resources/Image/btn_Return_press.png);}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_title">
           <property name="minimumSize">
            <size>
             <width>504</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>504</width>
             <height>71</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">
min-width:504px;
max-width:504px;
min-height:71px;
max-height:71px;
border-image: url(:/Image/Resources/Image/请扫描二维码存件.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_home">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">#btn_home
{
min-width:55px;
max-width:55px;
min-height:50px;
max-height:50px;
border-radius: 10px;
}
#btn_home{border-image: url(:/Image/Resources/Image/btn_home_nor.png);}
#btn_home:hover{border-image:url(:/Image/Resources/Image/btn_home_sele.png);}
#btn_home:pressed{border-image:url(:/Image/Resources/Image/btn_home_press.png);}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_gif">
          <property name="minimumSize">
           <size>
            <width>325</width>
            <height>326</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>325</width>
            <height>326</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtControl.qrc"/>
 </resources>
 <connections/>
</ui>
